classes/c.oeclass

```diff
@@ -4,6 +4,8 @@
 ## machine, sdk, cross, sdk-cross, canadian-cross (=all). A similar class
 ## exists for c++, see the c++ class.
 
+CLASS_FLAGS:>machine	= " powerpc_32bit_addr"
```

Comment:
It should probably be a '+=' in case it is combined with another class that uses CLASS_FLAGS:>machine.